src/__tests__/vuex.js

```diff
@@ -26,7 +32,7 @@ test('can render with vuex with defaults', async () => {
 
 test('can render with vuex with custom initial state', async () => {
   const {getByTestId, getByText} = renderVuexTestComponent({
-    state: {count: 3},
+    state: () => ({count: 3}),
```

Comment:
So `state` accepts a function similar to `data` now? (Please excuse the noob questions. :sweat_smile:)

Reply:
This is actually not related to the change of this PR, so I'll revert to prevent any confusion.